easybuild/easyconfigs/m/MedPy/MedPy-0.4.0-foss-2019b-Python-3.7.4.eb

```diff
@@ -0,0 +1,34 @@
+# This easyconfig was created by the BEAR Software team at the University of Birmingham.
+easyblock = 'PythonBundle'
+
+name = 'MedPy'
+version = '0.4.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = "https://pypi.org/project/MedPy/"
+description = """MedPy is a library and script collection for medical image processing in Python, providing basic
+ functionalities for reading, writing and manipulating large images of arbitrary dimensionality. Its main
+ contributions are n-dimensional versions of popular image filters, a collection of image feature extractors, ready
+ to be used with scikit-learn, and an exhaustive n-dimensional graph-cut package."""
+
+toolchain = {'name': 'foss', 'version': '2019b'}
+
+dependencies = [
+    ('Python', '3.7.4'),
+    ("SimpleITK", "1.2.4", versionsuffix),
```

Comment:
We usually use single quotes for string values that don't have spaces (see line above)